recipes/CNTK-GPU-OpenMPI/docker/run_convnet_mnist_gpu.sh

```diff
@@ -17,7 +17,7 @@ IFS=',' read -ra HOSTS <<< "$AZ_BATCH_HOST_LIST"
 nodes=${#HOSTS[@]}
 
 # special path for non-mpi job with single gpu
-if [ $nodes -eq 0 ] && [ $ngpus -eq 1 ]; then
+if [ $nodes -eq 1 ] && [ $ngpus -eq 1 ]; then
```

Comment:
The change will fail in the case, where a user creates a multi-instance task with number of instances set to number of vms in the pool, and starts with pool size 1, and wishes to expand later on to more nodes after some validation. In this case, the script will end up treating it as a single instance task. Do we care about this scenario?